src/theory/quantifiers/sygus/cegis.cpp

```diff
@@ -166,7 +166,17 @@ bool Cegis::constructCandidates(const std::vector<Node>& enums,
     Trace("cegis") << "  Enumerators :\n";
     for (unsigned i = 0, size = enums.size(); i < size; ++i)
     {
-      Trace("cegis") << "    " << enums[i] << " -> ";
+      bool isUnit = false;
```

Comment:
isUnit is just for tracing purposes? If so, can you guard this is Trace.isOn("cegis")?

Reply:
This code block in already inside a `Trace.isOn("cegis")` block